docs/api-docs/channels/channels-extending-existing.md

```diff
@@ -163,8 +163,22 @@ Accept: application/json
 | `acquia`          | `storefront`              |
 | `bloomreach`      | `storefront`              |
 | `deity`           | `storefront`              |
-| `google_shopping` | `marketing`               |
-| `custom`          | `storefront`, `pos`, `marketing`, `marketplace`             |
+| `google_shopping`\* | `marketing` |
+| `google` | `marketing` |
+| `google` | `marketplace` |
+| `custom` | `storefront`, `pos`, `marketing`, `marketplace` |
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+>
+> \* The `google_shopping` platform is depreciated and should **not** be used. Support for the types `marketing` and `marketplace` on the `google` platform are being added.

```

Comment:
We discourage using  `google_shopping`  as BigCommerce has deprecated this platform. Instead, we have added support for `marketing` and `marketplace` types on the `google` platform.

Reply:
I wanted to replace the passive voice.

Reply:
Thanks, I made those changes.